includes/class-wc-gutenberg-emails-loader.php

```diff
@@ -13,6 +13,42 @@ class WC_Gutenberg_Emails_Loader {
 	 * Constructor.
 	 */
 	public function __construct() {
+		add_action( 'init', array( $this, 'init_post_type' ) );
+		add_action( 'init', array( $this, 'create_emails' ) );
+	}
+
+	/**
+	 * Init post types
+	 */
+	public function init_post_type() {
+		register_post_type(
+			'woocommerce-emails',
+			array(
+				'labels'           => array(
+					'name'          => __( 'WooCommerce Emails', 'woocommerce-gutenberg-emails' ),
+					'singular_name' => __( 'WooCommerce Email', 'woocommerce-gutenberg-emails' ),
+				),
+				'public'           => false,
```

Comment:
I had another thought– we can use `'menu_position' => 56` to put this in the WooCommerce section, then in `labels`, add `'menu_name' => __( 'Emails', 'woocommerce-gutenberg-emails' )` – now the name doesn't wrap 🙂